content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -34,8 +34,13 @@ The operator is beta in 1.6 and suitable for production use.
     - A service to access operator metrics
     - Necessary Istio operator RBAC rules
 
-    See the available `istioctl operator init` flags to control which namespaces the controller and Istio are installed
-    into and the installed Istio image sources and versions.
+    You can also pass single or multiple namespaces to watch by using `--watchedNamespaces` flag.
+
+    {{< text bash >}}
+    $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2
```

Comment:
Why is this flag not documented? https://preliminary.istio.io/latest/docs/reference/commands/istioctl/#istioctl-operator-init

Reply:
i think it's because the feature is not implemented yet - see PR description.

Reply:
Users are confused with the term `istioNamespace` see this https://github.com/istio/istio/issues/24459  issue, if the user provides this flag, they assume that the Istio will be installed in the provided namespace. But the reality is that the `--istioNamespace` flag in `operator init` is used to watch the namespace. 

To avoid this confusion @morvencao has added `--watchedNamespaces` which clearly states that it can watch multiple namespaces, Documenting it will really help to avoid confusion. See more details https://github.com/istio/istio/pull/24696.

Reply:
Thanks @shamsher31 
`--watchedNamespaces` is more clear than `--istioNamespace`,  the namespace where istio components are installed should be defined in IOP instance, should not be specified in `operator init` command.

Reply:
This whole thing (separate namespaces for 1. running operator, 2. CRDs that operator responds to, 3. istio is installed in) seems like an overly complicated design. What happens if there are multiple CRDs in different watched namespaces with different configurations of Istio in istio-system? Why not require that 2 and 3 are the same, and watchedNamespaces would be "all namespaces"? Then you only need one flag in `istioctl operator init` to specify the namespace to run operator.